exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+let greeting = "Hello, "
+let myname = "My name is Shima"
+let message = greeting + myname;
 console.log(message);
```

Comment:
Fantastic work, this does the job, again it's just changing `let` to `const` since you never change the values of each variable. 👍